ruby/spec/cucumber/ci_environment/variable_expression_evaluate_spec.rb

```diff
@@ -1,38 +1,40 @@
+# frozen_string_literal: true
+
 require 'cucumber/ci_environment'
 
-describe 'Cucumber::CiEnvironment::VariableExpression.evaluate' do
-  include Cucumber::CiEnvironment::VariableExpression
+describe Cucumber::CiEnvironment::VariableExpression, '#evaluate' do
+  include described_class
 
   it 'returns nil when a variable is undefined' do
-    expression = "hello-${SOME_VAR}"
+    expression = 'hello-${SOME_VAR}'
     result = evaluate(expression, {})
-    expect(result).to eq(nil)
+    expect(result).to be_nil
   end
 
   it 'gets a value without replacement' do
-    expression = "${SOME_VAR}"
+    expression = '${SOME_VAR}'
     result = evaluate(expression, { 'SOME_VAR' => 'some_value' })
     expect(result).to eq('some_value')
   end
 
   it 'captures a group' do
-    expression = "${SOME_REF/refs\/heads\/(.*)/\\1}"
+    expression = '${SOME_REF/refs/heads/(.*)/\\1}'
     result = evaluate(expression, { 'SOME_REF' => 'refs/heads/main' })
     expect(result).to eq('main')
   end
 
   it 'works with star wildcard in var' do
-    expression = "${GO_SCM_*_PR_BRANCH/.*:(.*)/\\1}"
+    expression = '${GO_SCM_*_PR_BRANCH/.*:(.*)/\\1}'
     result = evaluate(expression, { 'GO_SCM_MY_MATERIAL_PR_BRANCH' => 'ashwankthkumar:feature-1' })
     expect(result).to eq('feature-1')
   end
 
   it 'evaluates a complex expression' do
-    expression = "hello-${VAR1}-${VAR2/(.*) (.*)/\\2-\\1}-world"
+    expression = 'hello-${VAR1}-${VAR2/(.*) (.*)/\\2-\\1}-world'
     result = evaluate(expression, {
```

Comment:
If you're comfortable using multi-line here this can be fixed. Or I can tackle it at a later date

Reply:
Sorry, I'm not sure what is meant by "multi-line" here

Reply:
This stylistically is quite wild now, so ideally it should be on multiple lines. i.e.

```
(
  {
  }
)
```

If you're not comfortable tidying it up, don't feel the need to fix it all. We can revisit these at a later date.

Reply:
I guess I'm not understanding how the highlighted line could be refactored into multi line. The rest of that block is multi line. Do you mean the other blocks in this file?

Reply:
As written above, the rubocop parser is detecting you want to have this right-aligned, which shifts all the code along. If you make it left aligned in the braces then it fixes this issue